tutorials/how-to/_posts/2019-02-10-sell-digital-art-online.md

```diff
@@ -94,13 +94,13 @@ This is just an example that generates files like this:
 
 Now that we have an image file fit for printing, we're going to use a service called [Printful](https://www.printful.com/) to print and ship our physical products.
 
-Register for a Printful account [here](https://www.printful.com/auth/register).
+[Register for a Printful account here](https://www.printful.com/auth/register).
 
 ## Order a Sample
 
 In the next steps we're going to create an online store that will allow us to sell our art to customers, but before we get there, I'd recommend ordering a couple things from Printful yourself.
 
-Go [here](https://www.printful.com/dashboard/order/update) to create an order. The Printful workflow generally works like this:
+[Creat a printful order](https://www.printful.com/dashboard/order/update). The Printful workflow generally works like this:
```

Comment:
Can you change this to "Create a Printful order"

(Add an "e" to "create" and capitalize "Printful")